mozetl/experimentsdaily/rollup.py

```diff
@@ -0,0 +1,73 @@
+import datetime as DT
+from pyspark.sql import SparkSession, functions as F
+import click
+from mozetl.clientsdaily.rollup import extract_search_counts
+from mozetl.utils import format_spark_path
+from fields import EXPERIMENT_FIELD_AGGREGATORS
+
+LAG = 10
+ACTIVITY_DATE_COLUMN = F.expr(
+    "substr(subsession_start_date, 1, {})".format(LAG)
+).alias("activity_date")
+EXPERIMENTS_SUMMARY_PATH = 's3://telemetry-parquet/experiments/v1/'
+EXCLUDED_ID = 'pref-flip-screenshots-release-1369150'
+ACTIVITY_SUBMISSION_LAG = DT.timedelta(LAG)
+WRITE_VERSION = '2'
+STORAGE_BUCKET = 'net-mozaws-prod-us-west-2-pipeline-analysis'
+STORAGE_PREFIX = '/spenrose/experiments-daily/v{}/'.format(WRITE_VERSION)
+
+
+def load_experiments_summary(spark, parquet_path):
+    return (
+        spark
+        .read
+        .option("mergeSchema", "true")
+        .parquet(parquet_path)
+        .where("experiment_id != '{}'".format(EXCLUDED_ID))
```

Comment:
Should you also exclude all rows/clients that are not enrolled in an experiment?

Reply:
s3://telemetry-parquet/experiments/v1/ is all main_summary rows that are not in an experiment.

Reply:
It's the route for the ExperimentSummaryView ([telemetry.experiments sample query](https://sql.telemetry.mozilla.org/queries/39852/source)), which is all the main_summary rows that are in an experiment, grouped by experiment and submission date.

Does the output of this script keep clients that do not have an `experiment_id`?